toolbox/src/main/AndroidManifest.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
     package="com.asf.appcoins.toolbox">
-
+  <uses-permission android:name="android.permission.INTERNET" />
```

Comment:
Do we need this permission here when it's set up already in the modules imported that have the permission set up?